source/crud/tests/server_2_6/read/aggregate.yml

```diff
@@ -0,0 +1,28 @@
+data:
+    - {_id: 1, x: 11}
+    - {_id: 2, x: 22}
+    - {_id: 3, x: 33}
+serverVersion: '2.6'
```

Comment:
Since we discussed denoting that this is a required minimum version, @xdg suggested renaming this field to `minServerVersion`. We have no use for `maxServerVersion` yet, but I suppose both could be documented in the README.

Keeping the lower/upper inclusive bounds as simple values will make it easier for the less dynamic languages to parse.